qa/remote-clusters/docker-compose.yml

```diff
@@ -30,6 +30,7 @@ services:
        - xpack.http.ssl.verification_mode=certificate
        - xpack.security.transport.ssl.verification_mode=certificate
        - xpack.license.self_generated.type=trial
+       - action.support.DestructiveOperations.REQUIRES_NAME_SETTING=false
```

Comment:
This should be `action.destructive_requires_name=false`, here and in other `docker-compose.yml` files. Making this change should take care of the unknown setting failure message.

Reply:
Done.